setup.py

```diff
@@ -32,6 +35,16 @@
 tf_cpu = 'tensorflow'
 tf_gpu = 'tensorflow-gpu'
 
+
+def latest_version(package_name):
+    url = "https://pypi.python.org/pypi/%s/json" % (package_name,)
+    data = json.load(request.urlopen(url))
+    # filter out rc and b releases and, more in general, any realeses that do not contain exclusively numbers and dots.
+    versions = [v for v in data["releases"].keys() if re.match('[0-9\.]+$',v)]  
```

Comment:
```suggestion
    versions = [v for v in data["releases"].keys() if re.match("[0-9.]+$", v)]  
```
note that in a regexp, a `.` inside `[]` counts as a *literal* period, not as "any character".